src/main/java/de/nplay/moderationbot/permissions/BotPermissionsCommands.java

```diff
@@ -0,0 +1,131 @@
+package de.nplay.moderationbot.permissions;
+
+import com.github.kaktushose.jda.commands.annotations.Inject;
+import com.github.kaktushose.jda.commands.annotations.interactions.*;
+import com.github.kaktushose.jda.commands.data.EmbedCache;
+import com.github.kaktushose.jda.commands.dispatching.interactions.commands.CommandEvent;
+import com.github.kaktushose.jda.commands.dispatching.interactions.components.ComponentEvent;
+import de.nplay.moderationbot.embeds.EmbedColors;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.interactions.components.selections.SelectMenu;
+
+import java.util.List;
+
+@Interaction
+public class BotPermissionsCommands {
+
+    @Inject
+    private EmbedCache embedCache;
+
+    private Member targetMember;
+    private Role targetRole;
+
+    @SlashCommand(value = "permissions list", desc = "Zeigt die Berechtigungen eines Benutzers an", isGuildOnly = true, enabledFor = Permission.BAN_MEMBERS)
+    @Permissions(BotPermissionFlags.PERMISSION_READ)
+    public void onPermissionsList(CommandEvent event, @Optional Member member) {
+        Member target = member == null ? event.getMember() : member;
+
+        assert target != null;
```

Comment:
Using assert ist bad practice in Java

Reply:
How would you solve this? With Object.requireNonNull()?

Reply:
Yeah, but IntelliJ is marking this as an error, since Event#getMember is Nullable
Or do we just want to suppress the error?

Reply:
Theoretically it is nullable, but as stated before in our case its never null, see the JDA docs: 
```
The Member who caused this interaction. This is null if the interaction is not from a guild.
```
jda-commands ensures that this interaction is _always_ from a guild. So either ignore the IntelliJ warning or supress it